localization/tr_TR.inc

```diff
@@ -0,0 +1,42 @@
+<?php
+// Labels used for different portions of the plugin
+$labels = array();
+$labels['activate'] = 'Aktif Et';
+$labels['twofactor_gauthenticator'] = 'İki Faktörlü Doğrulama (2FA)';
+$labels['code'] = 'Google Authenticator Kodu';
+
+$labels['two_step_verification_form'] = '2-Faktörlü Doğrulama Kodu:';
+
+$labels['secret'] = 'Gizli Kod';
+$labels['qr_code'] = 'QR Kodu';
+$labels['msg_infor'] = 'İki faktörlü doğrulama için Android veya IOS İşletim sistemli telefonunuza Google Authenticator Uygulaması indirmeniz gereklidir.<br>
+IOS : <a href="https://apps.apple.com/tr/app/google-authenticator/id388497605" target="_blank">Uygulamaya git</a><br>
+Android : <a href="https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2&hl=en_US" target="_blank">Uygulamaya git</a>';
+
+$labels['show_secret'] = 'Gizli Kodu Göster';
+$labels['hide_secret'] = 'Kodu Gizle';
+$labels['create_secret'] = 'Giz kod oluştur';
+
+$labels['show_qr_code'] = 'QR Kodu Göster';
+$labels['hide_qr_code'] = 'QR Kodu Gizle';
+
+$labels['recovery_codes'] = 'Kurtarma Kodu';
+$labels['show_recovery_codes'] = 'Kurtarma Kodunu Göster';
+$labels['hide_recovery_codes'] = 'Kurtarma Kodunu Gizle';
+
+$labels['setup_all_fields'] = 'Tüm alanları doldurun (ayarlarınızı saklamak için kaydet'e tıkladığınızdan emin olun)';
```

Comment:
$labels['setup_all_fields'] = 'Tüm alanları doldurun (ayarlarınızı saklamak için kaydet&#39;e tıkladığınızdan emin olun)';